src/core/config.h

```diff
@@ -39,6 +39,7 @@ typedef enum {
     CONFIG_GP_CH_WAREHOUSES_DONT_ACCEPT,
     CONFIG_GP_CH_HOUSES_DONT_EXPAND_INTO_GARDENS,
     CONFIG_GP_CH_MONUMENTS_BOOST_CULTURE_RATING,
+    CONFIG_GP_CH_DISABLE_INFINITE_WOLVES_SPAWNING,
```

Comment:
To ensure that a new config option is properly saved and loaded you need to couple it with an .ini key in core/config.c Otherwise the game will crash on start.

Reply:
Done. In the same order as in `config.h`, which is after `gameplay_change_monuments_boost_culture_rating`